corehq/apps/ota/tests/test_search_claim_endpoints.py

```diff
@@ -86,6 +89,7 @@ def test_duplicate_claim(self):
 
     @run_with_all_backends
     def test_search_endpoint(self):
+        # NOTE: Requires `ALLOW_FORM_PROCESSING_QUERIES = True` in localsettings.py
```

Comment:
That comment was true for a while, when I was trying to find a way to use CommCareCaseSQLSerializer to instantiate cases from results. It's not true any more: #11577